crazyflie/scripts/crazyflie_server.py

```diff
@@ -427,7 +427,7 @@ def _log_scan_data_callback(self, timestamp, data, logconf, uri):
             right_range = float("inf")
         if back_range > max_range:
             back_range = float("inf")  
-        self.ranges = [back_range, left_range, front_range, right_range, back_range]
```

Comment:
I think I had a reason that i did this, as it worked better in the slamtoolbox, but I'll try it out soon. It is cleaner to remover the second back range.

Reply:
Should I remove this change from this PR, then (and/or mimick the cflib behavior in the cpp backend instead)?

Reply:
no this is fine, I just have to remind myself to test if all the examples are still working.